app/mocks.ts

```diff
@@ -78,3 +78,24 @@ export const colors = [
   "bg-cyan-600",
   "bg-sky-600",
 ];
+
+
+export const currencyToCountryCode: Record<string, string> = {
+  KES: "ke",
+  NGN: "ng",
+  GHS: "gh",
+  BRL: "br",
+  ARS: "ar",
+  USD: "us",
+  EUR: "eu",
+  GBP: "gb",
+  ZAR: "za",
+  UGX: "ug",
+  TZS: "tz",
+  RWF: "rw",
+  INR: "in",
+  CAD: "ca",
+  AUD: "au",
+  JPY: "jp",
+  CNY: "cn",
+};
```

Comment:
Instead of a static mapping, we should extract the country code dynamically from the currency code. Most ISO 4217 currency codes start with a two-letter country code:

KES → KE (Kenya)
NGN → NG (Nigeria)
USD → US (United States)

We can create a small override map if there are exceptions and use `currencyCode.substring(0, 2).toLowerCase()` as the default logic.